mlir/lib/Dialect/MIOpen/MIOpenOps.cpp

```diff
@@ -59,6 +59,39 @@ static void print(OpAsmPrinter &p, Conv2DOp op) {
 }
 
 static LogicalResult verify(Conv2DOp op) {
+
+  {
```

Comment:
please remove anonymous blocks. try use lambdas in case you might have repeated logic.